roles/zram/main.yml

```diff
@@ -20,10 +20,13 @@
     name: "{{ item.name }}"
     value: "{{ item.value }}"
   with_items:
+    # https://docs.kernel.org/admin-guide/sysctl/vm.html#swappiness
     - { key: "vm.swappiness", value: "120" }
-    - { key: "vm.watermark_boost_factor", value: "0" }
-    - { key: "vm.watermark_scale_factor", value: "125" }
-    - { key: "vm.page-cluster", value: "0" }
+    #- { key: "vm.watermark_boost_factor", value: "0" }
+    #- { key: "vm.watermark_scale_factor", value: "125" }
+    #- { key: "vm.defrag_mode", value: "1" }
```

Comment:
it may still be worth experimenting with some of these options, particularly defrag_mode